tests/sklearn/test_sklearn_autolog.py

```diff
@@ -1806,3 +1808,24 @@ def test_autolog_emits_warning_message_when_pos_label_used_for_multilabel():
             "Metric error: Target is multiclass but average='binary'. Please choose another "
             "average setting, one of [None, 'micro', 'macro', 'weighted']."
         )
+
+
+def test_sklearn_distutils_issue(monkeypatch, tmp_path):
```

Comment:
The fix is pending. We must ensure this test really fails.

Reply:
Interesting. I could not repro the issue on CI.

Reply:
Failed! 
https://github.com/mlflow/mlflow/actions/runs/10192119881/job/28194484685?pr=12847

Reply:
It seems it's fixed on sklearn master branch.